store/index.js

```diff
@@ -178,6 +192,22 @@ export const getters = {
         return nextPage;
     },
 
+    getCategoricalOptions: (p_state) => (p_column) => {
+
+        // Return the options for this column listed in the current (hardcoded)
+        // options for each categorical data-based category
+        return p_state.categoricalOptions[p_state.columnToCategoryMapping[p_column]] ?? [];
```

Comment:
wow, nice syntax thing. Learned something.

Reply:
I figured we'd give it a try here. Maybe convert some other statements to similar syntax at a later date!